src/runtime/components/forms/Form.vue

```diff
@@ -87,7 +89,7 @@ export default defineComponent({
         if (errors) {
           errs = errs.concat(errors)
         } else {
-          Object.assign(props.state, result)
```

Comment:
@romhml I think we shouldn't mutate the `state` data directly because when we use transformation methods like `.transform((v) => v.toUpperCase())`, it automatically changes the input value displayed to the user. Instead, the transform method should only modify the return value while preserving the original state

Reply:
Ok for me, this can be done in the `onSubmit`  callback if needed anyway.